src/Behavioral.Automation/Bindings/ListBinding.cs

```diff
@@ -24,8 +24,17 @@ public void CheckListContainsItems(IListWrapper list, string behavior, Table tab
             var expectedListValues = ListServices.TableToRowsList(table);
 
             bool exactOrder = behavior.Contains("contain in exact order");
-            bool check = list.ListValues.ContainsValues(expectedListValues, exactOrder);
-            check.Should().Be(!behavior.Contains("not contain"));
+
+            bool checkResult;
+            if (behavior.StartsWith("not"))
+            {
+                checkResult = list.ListValues.DoesntContainValues(expectedListValues);
```

Comment:
we should check that list doesn't contain all listed values, so previous approach wasn't correct